src/ripple/core/Stoppable.h

```diff
@@ -209,6 +209,16 @@ class Stoppable
     /** Destroy the Stoppable. */
     virtual ~Stoppable ();
 
+    RootStoppable& getRoot() {return m_root;}
```

Comment:
should we just go ahead and make `m_root` private?

Reply:
Not sure what you mean, `m_root` is private.

Reply:
you're right - please ignore.